dashboard/app/controllers/api/v1/census/census_controller.rb

```diff
@@ -104,6 +105,7 @@ def create
       census_params[field] = census_params[field].strip_utf8mb4 unless census_params[field].nil?
     end
 
+    census_params[:school_year] = current_census_year
```

Comment:
Where the census year is being set (using `census_helper.rb`)

Reply:
looks like it is set like this: https://github.com/code-dot-org/code-dot-org/blob/42daea7a8de11c8d9d64ece5946e16db2fd5c54a/lib/census_helper.rb#L5

good enough for now! I think it would be worth following up with the RED team to see if they even care about this field. it's possible they could just use the created_at date of the submission, or that they don't even care about the date at all. if the DO care about it, then we should confirm whether this is the logic they want -- we might want a hard-coded constant instead so that we control when it gets updated (whenever submissions for the current year are closed). 

@TurnerRiley can you please add a Jira item to hard-code or remove the year from this model in our backlog, and we can discuss at our next meeting?

Reply:
Ah, I missed this! Thanks for pointing this out

Reply:
To close this out, this discussion is tracked in [this thread](https://codedotorg.slack.com/archives/C0SUN2W3D/p1677519796649959) where it was decided to keep the logic and set the `school_year` field but update the logic to match other definitions that RED has.